komorebi/settings.py

```diff
@@ -0,0 +1,149 @@
+import logging
+import os
+from enum import Enum
+
+from gi.repository import Gio, GLib
+
+import komorebi.consts
+
+key_file_group : str = 'KomorebiProperties'
+
+# Settings
+wallpaper_name : str = 'foggy_sunny_mountain'
+time_twenty_four : bool = True
+autostart : bool = True
+show_desktop_icons : bool = True
+enable_video_wallpapers : bool = True
+mute_playback : bool = False
+pause_playback : bool = True
```

Comment:
Uh, global variables are not great.

Reply:
Fixed; it's not perfect still, but these variables are now inside a `Settings` class.